data/la/committees/lower-Administration-of-Criminal-Justice-f2a99f30-ffd7-4dfb-90db-557ae74839e8.yml

```diff
@@ -0,0 +1,49 @@
+id: ocd-organization/f2a99f30-ffd7-4dfb-90db-557ae74839e8
+jurisdiction: ocd-jurisdiction/country:us/state:la/government
+classification: committee
+name: Administration of Criminal Justice
+chamber: lower
+sources:
+- url: https://www.legis.la.gov/legis/Committees.aspx?c=H
+  note: Committees List Page
+- url: https://house.louisiana.gov/H_Cmtes/CriminalJustice.aspx
+  note: Committee Detail Page
+links:
+- url: https://house.louisiana.gov/H_Cmtes/CriminalJustice.aspx
+  note: homepage
+members:
+- name: Tony Bacala
+  role: Vice Chair
+  person_id: ocd-person/9fde5725-d1b2-4afc-9897-2706dda502c8
+- name: Bryan Fontenot
+  role: Member
+  person_id: ocd-person/90b6f6c3-9620-4fbb-8c23-f8ab7b163de7
+- name: Raymond E. Garofalo
+  role: Member
+  person_id: ocd-person/b3de3921-3132-43e4-b22b-5b907a21108e
+- name: I, Jonathan Goudeau
```

Comment:
Names with suffix are all like this, may want to update manually then I think you can run the cli lint with `--fix` & it should try to rematch so you don't have to manually?